server/controller/passportControllers.js

```diff
@@ -53,21 +52,24 @@ passport.use(
             }
 
             try {
-                const user = new User({
-                    username: username,
-                    githubProfile: profile,
-                    accounts: [
-                        {
-                            kind: 'github',
-                            uid: profile.id.toString(),
-                            token: {
-                                accessToken: accessToken
-                                // No refershToken given with github
+                const user = new User(
+                    {
+                        username: username,
+                        githubProfile: profile,
+                        accounts: [
+                            {
+                                kind: 'github',
+                                uid: profile.id.toString(),
+                                token: {
+                                    accessToken: accessToken
+                                    // No refershToken given with github
+                                }
                             }
-                        }
-                    ],
-                    commits: 0
-                });
+                        ],
+                        commits: 0
+                    },
+                    { _id: 0 }
```

Comment:
```suggestion
                    }
```

I realised I have not tested this part, when it's saving for new User. @darshkpatel could you let me know if you have any idea if this would work? 

If not, I'll remove `{ _id: 0}`, and work on it on a new issue sometime

Reply:
I have no idea

Reply:
Cool, time to tackle it in new issue then